scripts/patterns-reorg/models.py

```diff
@@ -96,6 +96,12 @@ def entries_as_markdown_list(
     return "\n".join(result)
 
 
+@dataclass(frozen=True)
+class DeletedPage:
+    slug: str
+    from_file: str
```

Comment:
I recommend calling it `old_slug` and `redirect_to`

Reply:
I think the names you suggest are better, but in this case, it is convenient to have the same names as the class `Entry` to avoid having to check more things in the `determine_redirects` function. With the same names, we can treat them as `Entry`s indistinctly, and all is more general. If you have a strong opinion about it, I can change it though

Reply:
Yeah, I would still change it. `from_file` is a misnomer because you would need the file extension set. It would be better for the code paths to be distinct in the for loop